plugins/modules/vmware_guest_cross_vc_clone.py

```diff
@@ -219,12 +223,21 @@ def __init__(self, module):
         self.destination_vcenter_password = self.params['destination_vcenter_password']
         self.destination_vcenter_port = self.params.get('port', 443)
         self.destination_vcenter_validate_certs = self.params.get('destination_vcenter_validate_certs', None)
+        self.timeout = self.params.get('timeout')
 
     def get_new_vm_info(self, vm):
         # to check if vm has been cloned in the destination vc
+        # connect to destination VC
         # query for the vm in destination vc
         # get the host and datastore info
         # get the power status of the newly cloned vm
+        self.destination_content = connect_to_api(
+            self.module,
+            hostname=self.destination_vcenter,
+            username=self.destination_vcenter_username,
+            password=self.destination_vcenter_password,
+            port=self.destination_vcenter_port,
+            validate_certs=self.destination_vcenter_validate_certs)
```

Comment:
Why do you connect to the destination vCenter again? Because the original connection timed out if the cloning takes, say two hours?

Just want to understand, basically the PR looks OK to me.